sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/tests/JObjectTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub.Tests
+{
+    public class JObjectTests
+    {
+        public static IEnumerable<object[]> MessageTestData =>
+            new List<object[]>
+            {
+                new object[] {new WebPubSubMessage("Hello"), MessageDataType.Binary, "Hello" },
+                new object[] {new WebPubSubMessage("Hello"), MessageDataType.Json, "Hello" },
+                new object[] {new WebPubSubMessage("Hello"), MessageDataType.Text, "Hello" },
+                new object[] {new WebPubSubMessage(Encoding.UTF8.GetBytes("Hello")) , MessageDataType.Binary, "Hello" },
+                new object[] {new WebPubSubMessage(Encoding.UTF8.GetBytes("Hello")), MessageDataType.Json, "Hello" },
+                new object[] {new WebPubSubMessage(Encoding.UTF8.GetBytes("Hello")), MessageDataType.Text, "Hello" },
+                new object[] {new WebPubSubMessage(new MemoryStream(Encoding.UTF8.GetBytes("Hello"))), MessageDataType.Binary, "Hello" },
+                new object[] {new WebPubSubMessage(new MemoryStream(Encoding.UTF8.GetBytes("Hello"))), MessageDataType.Json, "Hello" },
+                new object[] {new WebPubSubMessage(new MemoryStream(Encoding.UTF8.GetBytes("Hello"))), MessageDataType.Text, "Hello" }
+            };
+
+        [Fact]
```

Comment:
Since this is a brand new extension library with a T2 SDK, can we use NUnit for tests instead?